docs/blog/newsletter-october-2020.rst

```diff
@@ -0,0 +1,125 @@
+
+A sensible way forward can be to combine both ideas. For example, providing a list of links to topics associated with a role; or allowing a user to filter a list of topics by role. It also depends a lot on your specific product, and the audience for it: role-based docs may work well for some products, while topic-based docs may be just the thing for others.
+
+-----------------------
+Command prompts in docs
+-----------------------
+
+Command prompts in docs can be useful visual cues for developers - they provide the context in which to run each command. They also show at a glance that the snippet displays a terminal command and not something else, like a config file. But to avoid mistakes during copying and pasting, our community has these suggestions on integrating the prompt character in documentation:
+
+* Do include prompts if the user modes change or are otherwise relevant to the instructions.
+* Do include prompts for multiple commands in the same snippet, but you might want to avoid prompts for single-line snippets.
+* Don’t include the command prompt in the same box as the output.
+* Make a clear distinction between the command and output. For example, by separating them into different listings with labels and distinguishing the prompt by color.
+* If you don’t want the prompt character to be copyable, try the user-select css property. Or, if you have control over its behaviour, have the copy button in the snippet ignore the prompt character.
+
+------------------------------------------
+What we're learning in #learn-tech-writing
+------------------------------------------
+
+Over in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ book club, they’re reading through *Don’t Make Me Think* by Steve Krug. One recent conversation was about documentation structure design: both for individual pages of documentation, and for doc sites too.
+
+At the page level, it’s about making pages scannable. You can break them up into clearly defined areas using headings, and it also helps to use short paragraphs and lists. One documentarian talked about using images to make pages look less daunting - while you want to avoid adding gratuitous images, they do really help break up pages visually.
+
+For a doc set, how do you make it easy for readers to find what they need? People search for information in different ways: some stick to Google or the site search, some scan and click around. It helps to talk to your users to find out, but you can’t expect them all to work the same way. And we have to be aware that our personal preferences don’t necessarily match those of our readers.
+
+A big point of discussion was on how to divvy up pages. At one extreme is individual short pages for each topic (About the Feature, Using the Feature, Reference for the Feature...). It can mean more clicking around, but DMMT suggests: "It doesn’t matter how many times I have to click as long as every click is a mindless, unambiguous choice."
```

Comment:
Should we spell out "DMMT" here since we didn't define the abbreviation in the first paragraph of this item?

Reply:
Yes, I was being lazy. Done!